lib/deas/server.rb

```diff
@@ -124,7 +124,7 @@ class Config
       attr_accessor :template_source, :logger, :router
 
       def initialize
-        @env             = DEFAULT_ENV
+        @env             = ENV['RACK_ENV'] || DEFAULT_ENV
```

Comment:
@kellyredding - I saw someone comment on `RACK_ENV` the other day and meant to look into it, saying it's not what most people think. This reminded me and I'm not sure we want to use it to default our env.

Apparently, `RACK_ENV` valid values are `development`, `deployment` or `none` and it controls what middlewares they add in by default:

* here's where `RACK_ENV` is used to default the environment option (https://github.com/rack/rack/blob/master/lib/rack/server.rb#L204)
* here's their comments on the environment option (https://github.com/rack/rack/blob/master/lib/rack/server.rb#L157)

So I don't think it's technically what we want to be setting the `RACK_ENV` to a rails like env `development`, `test`, `production`. We probably need to address this in some of our other stuff where we set `RACK_ENV` as well.